aspnetcore/fundamentals/metapackage.md

```diff
@@ -34,6 +33,18 @@ The following *.csproj* file references the `Microsoft.AspNetCore.All` metapacka
 
 [!code-xml[](metapackage/samples/Metapackage.All.Example.csproj?highlight=6)]
 
+The preceding markup represents a typical ASP.NET Core 2.0 template. It doesn't specify a version number for the `Microsoft.AspNetCore.All` package reference. When the version isn't specified, an [implicit](https://github.com/dotnet/core/blob/master/release-notes/1.0/sdk/1.0-rc3-implicit-package-refs.md) version is specified by the SDK (`Microsoft.NET.Sdk.Web`). We recommend relying on the implicit version specified by the SDK and not explicitly setting the version number on the package reference. If you have questions about this approach, leave a GitHub comment at the [Discussion for the Microsoft.AspNetCore.App implicit version](https://github.com/aspnet/Docs/issues/6430).
```

Comment:
I think we need 2.0 => 2.1. In 2.0, we expect explicit versions. Also I wouldn't link the dotnet release notes since the implicit reference to Microsoft.NETCore.App works quite a bit differently than the semi-implicit reference to Microsoft.AspNetCore.App.

Reply:
What I mean is that if we are adding a section regarding implicit versions, then it should be clear that we are talking about 2.1. There is no implicit versioning in 2.0. In fact I realized the sample apps are all mismatched right now with a netcoreapp2.0 tfm but using 2.1.* referenced.

Reply:
Yes, I see what you mean ... `2.0.9` would have been a better choice. :cry:

Reply:
Let me clarify a little here. In 2.0, all versions were explicit and the recommended way of targeting aspnetcore was through an explicit package reference to Microsoft.AspNetCore.All. In 2.1, the recommended way of targeting aspnetcore is through a version-less reference to Microsoft.AspNetCore.App. However, we still support references to Microsoft.AspNetCore.All in 2.1, though if you continue to use it, the recommendation would be to remove the version number and rely on the implicit version mechanisms.

Reply:
Looks like I should do the following:

* Restore the `Version` in the 2.0 `All`-based sample app project files but use `2.0.9`.
* Leave the current explanation alone and leave the example `netcoreapp2.0` with `All` and a version.
* Add a separate section here for the info on implicit versioning. Make it clear that it's for apps that target 2.1. Perhaps, include our usual recommendation (found elsewhere here, too) to use the `App` package. This section won't show actual code (i.e., `netcoreapp2.1` with `All`).

Reply:
Sounds good